js/src/client.ts

```diff
@@ -2091,16 +2117,18 @@ export class Client implements LangSmithTracingClientInterface {
         Object.entries(currentBody).filter(([_, value]) => value !== undefined)
       );
 
-      const response = await this.caller.call(_getFetchImplementation(), url, {
-        method: "POST",
-        headers: { ...this.headers, "Content-Type": "application/json" },
-        body: JSON.stringify(filteredPayload),
-        signal: AbortSignal.timeout(this.timeout_ms),
-        ...this.fetchOptions,
+      const response = await this.caller.call(async () => {
+        const res = await this._fetch(url, {
+          method: "POST",
+          headers: { ...this.headers, "Content-Type": "application/json" },
+          signal: AbortSignal.timeout(this.timeout_ms),
+          ...this.fetchOptions,
+          body: JSON.stringify(filteredPayload),
```

Comment:
Serialization happening inside `this.caller`

Reply:
Thanks fixing